### ISSUE TITLE
Block Placement Error

### ISSUE THREAD

**Issue by SwordOfSouls** [MMonkeyKiller/CustomBlocks#6]
**Describe the bug**
When someone places a custom block I get this error (the block still places):
https://pastebin.com/U7QgHzTS

**To Reproduce (if is possible)**
1. I am using a void world
2. Place a custom block

**Expected behavior**
For there not to be an error

**Screenshots**

**Server Info (please complete the following information):**
Paper: 1.18.1, 5e30e19

**Comment by SwordOfSouls** [comment]
Only happens when placing the block next to another block.

**Comment by SwordOfSouls** [comment]
As I needed the solution fast just made this override:
```
    @EventHandler
    public void place(BlockPlaceEvent event) {
        try {
            CustomBlock block = CustomBlock.getCustomBlockbyItem(event.getItemInHand());
            event.getBlock().getWorld().getBlockAt(event.getBlock().getLocation()).setType(Material.NOTE_BLOCK);
            NoteBlock note = (NoteBlock) event.getBlock().getWorld().getBlockAt(event.getBlock().getLocation()).getBlockData();
            event.getBlock().getWorld().getBlockAt(event.getBlock().getLocation()).setBlockData(block.getData().applyData(note));
            if (event.getItemInHand().getAmount() > 1)
                event.getItemInHand().setAmount(event.getItemInHand().getAmount() - 1);
            else {
                event.getItemInHand().getItemMeta().getPersistentDataContainer().set(new NamespacedKey(YoutuberBlocks.getInstance(), "ID"), PersistentDataType.STRING, UUID.randomUUID().toString());
                event.getPlayer().getInventory().remove(event.getItemInHand());
            }
            event.setCancelled(true);
        } catch (Throwable io) {
        }
    }
```

**Comment by MMonkeyKiller** [comment]
Hello! Is now solved in [3.0.1-SNAPSHOT](https://github.com/MMonkeyKiller/CustomBlocks/releases/tag/3.0.1-SNAPSHOT)
